### ISSUE TITLE
drep-state is making --drep-verification mandatory

### ISSUE THREAD

**Issue by CarlosLopezDeLara** [input-output-hk/cardano-cli#346]
The help, suggests that * Get the DRep state. If no DRep credentials are provided, return states for all
  of them.*, however using any of the `--drep-` flags is mandatory.

- If the query can return all drep when no flag is provided, then the flag should not be mandatory <- *most desired!!*

- If the query can only return state for individual dreps, then we need to amend the help

```
cardano-cli conway governance query drep-state --testnet-magic 4
Missing:
  ( --drep-verification-key STRING
  | --drep-verification-key-file FILE
  | --drep-key-hash HASH
  )
```

```
Usage: cardano-cli conway governance query drep-state
                                                        [--socket-path SOCKET_PATH]
                                                        [ --shelley-mode
                                                        | --byron-mode
                                                          [--epoch-slots SLOTS]
                                                        | --cardano-mode
                                                          [--epoch-slots SLOTS]
                                                        ]
                                                        ( --mainnet
                                                        | --testnet-magic NATURAL
                                                        )
                                                        ( --drep-verification-key STRING
                                                        | --drep-verification-key-file FILE
                                                        | --drep-key-hash HASH
                                                        )
                                                        [--out-file FILE]

  Get the DRep state. If no DRep credentials are provided, return states for all
  of them.
```

**Comment by carlhammann** [comment]
@CarlosLopezDeLara It seems that the same issue was also holds for `cardano-cli conway governance query drep-stake-distribution`. I fix both issues in #348.

**Comment by CarlosLopezDeLara** [comment]
> @CarlosLopezDeLara It seems that the same issue was also holds for `cardano-cli conway governance query drep-stake-distribution`. I fix both issues in #348.

Asewome!! Il give it a test drive shortly
